### DISCUSSION TITLE
Remove alonso from evac center?

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#3584] [Polls]
For context, please visit #3582.

The poll ends in [2023-11-09 23:59:59 anywhere on earth.](https://time.is/Anywhere_on_Earth)

**Comment by chaosvolt** [comment]
Personally I'm fine either way but leery of the comments likely to ensue during and after the PR's merged. If I'd had the time and energy to tackle NPC dialogue overhauls I would've favored cutting out the optional sexual encounter as the first step, and ideally rewriting his dialogue to be higher quality as a part a general quality pass, but:
1. I'm not the one to tackle it first.
2. Even with merging this, un-obsoleting him whenever I get around to said quality pass to rewrite his dialogue is always on the table anyway.

In lieu of a "I'm fine either way" option went with yes.

**Reply by Treah** [reply]
Is there some quest where your forced to interact with this char? If not why don't people just avoid him when they are playing. Does this really piss them off so much they need it gone to feel ok playing? I never understood why people get this worked up over a video game character if they do something they feel is cringe or controversial. There is plenty of other stuff in the game that is equally terrible such as killing people and eating them or turning them into funny named meals, or killing a bunch of zombie kids in a school with a automatic weapon but one guy that's horny and wants condoms is where you draw the line..... Seams a bit silly to me.  I do not give a crap about the char in general but I feel like if you start trying to tone down the game and removing content because it offends people your starting down a slippery slope of over PC garbage and pandering to cry baby people that cant function in adult society. 

This is my 2cents. 
Rework the char to have better dialog or just keep him there and people can I don't know, avoid talking to him.

**Reply by KheirFerrum** [reply]
My opinion is mainly, that it's badly written and while seasoned players can avoid him, he's currently part of the refugee center experience, and most new players who get into the game will almost certainly mosey their way over as the evac shelter specifically points you in it's direction.

Alonso is thus not really out of the way, and in my opinion gives a bad impression of what to expect from NPCs in the game. Furthermore, his entire purpose in the game is stated to be the failed town bicycle, trading condoms for morale, so reworking him would pretty much require throwing out that entire concept because it's a terrible concept, at which point my question is are we really so attached to Alonso as a character that enough of their interactions are worth salvaging over simply removing him entirely so we can place a new NPC in his place without the baggage he carries.

**Comment by Krwak** [comment]
> Is there some quest where your forced to interact with this char? If not why don't people just avoid him when they are playing. Does this really piss them off so much they need it gone to feel ok playing? I never understood why people get this worked up over a video game character if they do something they feel is cringe or controversial. There is plenty of other stuff in the game that is equally terrible such as killing people and eating them or turning them into funny named meals, or killing a bunch of zombie kids in a school with a automatic weapon but one guy that's horny and wants condoms is where you draw the line..... Seams a bit silly to me. I do not give a crap about the char in general but I feel like if you start trying to tone down the game and removing content because it offends people your starting down a slippery slope of over PC garbage and pandering to cry baby people that cant function in adult society.
> 
> This is my 2cents. Rework the char to have better dialog or just keep him there and people can I don't know, avoid talking to him.

I think that while a valid option, it's not a good way to resolve problems that you simply tell people to ignore them.
Players do talk with this character either by accident or not and they find the character 'cringy' at best, it ruining immersion is another word for it.
PRs for the game are trying to make the game better.
Would the game feel better without this character? I think that definitely yes.

Edit:  I would also like to point out that the very fact that this character is causing so much drama in the community is a reason enough.
Character is badly written, inappropriate and controversial on top of that. I don't think there is anything to salvage here.

**Comment by chaosvolt** [comment]
The main issues with the character are:
1. Poor quality of writing. This is common in a lot of Erk's NPCs in general. Ideally the better fix for that is to actually write better dialogue, but as mentioned that takes time and effort, and it's always possible to just hit him with the "better writing" stick later on and un-obsolete him, with this as a bandaid solution for now.
2. He's a flat character that mostly exists to be a nympho stereotype. I have yet to see anyone in the discord server say they're personally hurt or offended by that, but it has been remarked upon as a problem. Even if you don't see anything wrong with it, that does mean that game resources are being spent on a low-quality flat character that has no value outside of shock value. Fixing that the "proper" way will be a bit of work since that means rewriting him to not be a flat character, but really that just falls under the same general category of "write him more competently"
3. The optional sexual encounter, in theory, is perfectly fine since it's not explicit, and the player is the one that has to put effort into encountering it. However, I can see there being potential for trouble if someone raises a big enough stink over the fact that the player can start as an underage character. The fixes there would be either to just relegate sexual encounters to mods and let them worry about that, or else bother with coding in the ability to check for character age in NPC dialogue so we have the option of having him go "nope" in certain scenarios. Hell, for flavor having his reaction change if the player's super-mutated might be interesting too, but giving us a way out of potentially getting TOS'd would be the useful bit.

I do suspect that there will probably be people who're only in favor for "ew canon bi character, I should not even be allowed to encounter that even though you have to do a chain of quests to get in his pants" but people with those sorts of complaints existing does not invalidate the actual concerns of writing quality and other issues. It just mostly means I have to pay more attention to the discussion on here and in the discord and warn people if they get too heated or act sus about it. :V

If we see this followed up with people demanding the removal of the Borichenkos (who lack the baggage that comes with Alonso and are written more subtly) then we'll know who was likely arguing this proposal under less rational motives.

**Comment by Treah** [comment]
> The main issues with the character are:
> 
>     1. Poor quality of writing. This is common in a lot of Erk's NPCs in general. Ideally the better fix for that is to actually write better dialogue, but as mentioned that takes time and effort, and it's always possible to just hit him with the "better writing" stick later on and un-obsolete him, with this as a bandaid solution for now.
> 
>     2. He's a flat character that mostly exists to be a nympho stereotype. I have yet to see anyone in the discord server say they're personally hurt or offended by that, but it has been remarked upon as a problem. Even if you don't see anything wrong with it, that does mean that game resources are being spent on a low-quality flat character that has no value outside of shock value. Fixing that the "proper" way will be a bit of work since that means rewriting him to not be a flat character, but really that just falls under the same general category of "write him more competently"
> 
>     3. The optional sexual encounter, in theory, is perfectly fine since it's not explicit, and the player is the one that has to put effort into encountering it. However, I can see there being potential for trouble if someone raises a big enough stink over the fact that the player can start as an underage character. The fixes there would be either to just relegate sexual encounters to mods and let them worry about that, or else bother with coding in the ability to check for character age in NPC dialogue so we have the option of having him go "nope" in certain scenarios. Hell, for flavor having his reaction change if the player's super-mutated might be interesting too, but giving us a way out of potentially getting TOS'd would be the useful bit.
> 
> 
> I do suspect that there will probably be people who're only in favor for "ew canon bi character, I should not even be allowed to encounter that even though you have to do a chain of quests to get in his pants" but people with those sorts of complaints existing does not invalidate the actual concerns of writing quality and other issues. It just mostly means I have to pay more attention to the discussion on here and in the discord and warn people if they get too heated or act sus about it. :V
> 
> If we see this followed up with people demanding the removal of the Borichenkos (who lack the baggage that comes with Alonso and are written more subtly) then we'll know who was likely arguing this proposal under less rational motives.

Honestly this reply sounds very reasonable to me and was exactly my point. If the reason to remove is technical and shit writing then cool. If the real reason is "OMG this is cringe remove it" then no.
